server/app.js

```diff
@@ -5,10 +5,10 @@ var faker = require('faker');
 var mysql = require('mysql');
 var con = require('./database');
 
-app.use(express.static('public'));
+app.use(express.static(__dirname + '/../client/dist'));
 app.use(bodyParser.json());
 
-app.get('/', function(req, res) {
+app.get('/seed', function(req, res) {
```

Comment:
The seeding of your database should not be linked to a public route, and the code should live in a separate file. You will want to seed the database from the command line.

Reply:
Thanks Josh. I have made the appropriate changes.